src/vsl/parser/nodes/interfaceStatement.js

```diff
@@ -5,33 +5,41 @@ import Node from './node';
  * 
  */
 export default class InterfaceStatement extends Node {
-    
+
     /**
      * Constructs a generic function statement
      * 
      * @param {string[]} access - The access modifiers of the node
-     * @param {Identifier[]} superclass - The superclasses to inherit or implement
-     * @param {Node[]} statements - The interface's body.
+     * @param {Identifier} name - The name of the interface
+     * @param {Identifier[]} superclasses - The superclasses to inherit or implement
+     * @param {Node[]} statements - The interface's body
+     * @param {Annotation[]} annotations - The annotations of the interface
      * @param {Object} position - a position from nearley
      */
     constructor(
         access: string[],
         name: Identifier,
         superclasses: Identifier[],
-        statements: [],
+        statements: Node[],
+        annotations: Annotation[],
         position: Object
     ) {
         super(position);
-        
-        /** @type {string} */
+
+        /** @type {string[]} */
         this.access = access;
+        /** @type {Identifier} */
         this.name = name;
+        /** @type {Identifier[]} */
         this.superclasses = superclasses;
-        this.statements = statements;
+        /** @type {Node[]} */
+        this.statements = statements === null ? new CodeBlock([]) : statements;
```

Comment:
we shouldn't do `new CodeBlock([])` as `CodeBlock`'s constructor needs to take a `location` argument, without it things will bork pretty bad in terms of error handling/FIX-ITs/debugger.